kafka/producer.go

```diff
@@ -0,0 +1,48 @@
+package kafka
```

Comment:
I would suggest to give short comment to package

Reply:
fixed